features/environment.py

```diff
@@ -331,6 +337,8 @@ def from_environ(cls, config) -> "UAClientBehaveConfig":
 def before_all(context: Context) -> None:
     """behave will invoke this before anything else happens."""
     context.config.setup_logging()
+    logging.getLogger("botocore").setLevel(logging.INFO)
```

Comment:
Sorry, the intent here is to actually lower the verbosity. I often run the tests with `--log-level=debug` which triggers an inordinate amount of `boto` logging for aws tests. By setting `boto` to "info" then it stays quiet even when the root logger is at "debug".

I'll remove this and give it it's own PR - it might deserve something smarter like, "only set to info if the root level is debug"